bucket/miru.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "2.10.0",
+    "description": "Stream anime torrents, real-time with no waiting for downloads",
+    "homepage": "https://github.com/ThaUnknown/miru",
+    "license": "GPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ThaUnknown/miru/releases/download/v2.10.0/win-Miru-2.10.0.exe#/dl.7z",
+            "hash": "sha512:e3dd4bbbae71850da2424bef8c0b0b91ba0a81b35f84f0c155d3143481ea10ffc2a8a5f13d6e50416bf623db91b8d5aca6da73c66c12de2d0e5901c6cd1468de",
+            "installer": {
+                "script": "Expand-7zipArchive -Path \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\""
+            }
+        }
+    },
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Recurse -Force",
```

```suggestion
    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninstall*\" -Recurse -Force",
```

In official buckets, we only use `post_install` if we need to (e.g. modify registry, or install service, which requires pointing the path to `scoop\(APP NAME)\current`.

`post_install` works just fine though. It is a convention to use `pre_install` first.